tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -663,56 +665,31 @@ def _execute_test_suite(self, suite: TestSuiteInfo, test_context: AgentTestConte
                             summary.append(f"[Passed]  {test.name}")
                             log.info("******** [Passed] %s", test.name)
                             self._lisa_log.info("[Passed] %s", test_full_name)
-                            self._report_test_result(
-                                suite_full_name,
-                                test.name,
-                                TestStatus.PASSED,
-                                test_start_time)
+                            self._report_test_result(suite_full_name, test.name, TestStatus.PASSED, test_start_time)
```

Comment:
no changes in these except clauses, just collapsing the method calls to a single line for easier reading